test/inductor/test_cpu_select_algorithm.py

```diff
@@ -1909,27 +1909,6 @@ def forward(self, x):
             self.common(mod, (u,), atol=atol, rtol=rtol)
         self.assertEqual(counters["inductor"]["select_algorithm_autotune"], 1)
 
-    @patches
-    @torch.no_grad
-    @unittest.skipIf(not TEST_MKL, "Test requires MKL")
-    @parametrize("bs", (5,))
-    @parametrize("Mdim", (64,))
-    @dtypes(torch.float)
-    def test_bmm_self_square(self, bs, Mdim, dtype):
```

Comment:
This test is not needed now?

Reply:
I have added it back in, it required a change in `add_choices` to only return a single tensor when the input nodes are identical. I was thinking to add this in a follow up issue, but this fix was simpler than expected.